src/App.jsx

```diff
@@ -1,22 +1,26 @@
-
 import { BrowserRouter, Route, Routes } from "react-router-dom"
 import Nav from "./components/Nav/Nav"
-import HomePage from './components/HomePage/HomePage'
-import './App.css'
+import HomePage from "./components/HomePage/HomePage"
 import BookDetail from "./components/BookDetail/BookDetail"
+import ClaimedBooks from "./Components/HomePage/ClaimedBooks/ClaimedBooks"
+import AvailableBooks from "./components/HomePage/AllBooks/AvailableBooks/AvailableBooks"
+import "./App.css"
 
 function App() {
   return (
     <div>
       <BrowserRouter>
         <Nav />
+
         <Routes>
-          <Route path="/" element={<HomePage />} />
+          <Route path="/" element={<AvailableBooks />} />
```

Comment:
You now have 2 routes tht do the same thing, probably best to just keep this one and remove the duplicate (/books/available).